cmd/vela-worker/operate.go

```diff
@@ -23,6 +26,38 @@ func (w *Worker) operate() error {
 	if err != nil {
 		return err
 	}
+	// Define the database representation of the worker
+	// and register itself in the database
+	workerActive := true
+	workerLastCheckedIn := time.Now().Unix()
+	registryWorker := library.Worker{
+		Hostname:      &w.Config.Hostname,
+		Address:       &w.Config.Server.Address,
+		Routes:        &w.Config.Queue.Routes,
+		Active:        &workerActive,
+		LastCheckedIn: &workerLastCheckedIn,
+	}
```

Comment:
Thoughts on using our setters here?

This gives us less LOC and more closely follows how we write code for other parts of Vela 👍 

Also, we'll want to make sure that we're using the `UTC` format for the timestamps.

```suggestion
	registryWorker := new(library.Worker)
	registryWorker.SetHostname(&w.Config.Hostname)
	registryWorker.SetAddress(&w.Config.Server.Address)
	registryWorker.SetRoutes(&w.Config.Queue.Routes)
	registryWorker.SetActive(true)
	registryWorker.SetLastCheckedIn(time.Now().UTC().Unix())
```

Reply:
I've been thinking on this a little and I agree that consistency is best.

Reply:
Good catch on using UTC as well 😄